include/libassert/platform.hpp

```diff
@@ -4,6 +4,8 @@
 // Copyright (c) 2021-2024 Jeremy Rifkin under the MIT license
 // https://github.com/jeremy-rifkin/libassert
 
+#include <cassert>
```

Comment:
Very minor: Probably added by accident?

Reply:
Amazing typo: should be `<version>` :sweat_smile: